src/raster/r.area.createweight/r_area_createweight.html

```diff
@@ -0,0 +1,468 @@
+containing the numeric value to be used as a response variable (e.g.
+population count), specified via the <b>response_variable</b> parameter.
+
+<p>The user must specify at least one categorical raster map,
+<b>basemap_a</b>. Optionally, other variables can be added to the
+Random Forest model: A second categorical raster map can be defined
+with the parameter <b>basemap_b</b>, and/or a raster with continuous
+values (e.g. distance to the nearest road, school, hospital,
+university...) with the parameter <b>distance_to</b>. From these raster
+maps, the Random Forest's explanatory variables (i.e. covariates,
+predictors) are calculated.  For each of the categorical rasters
+(basemaps), the proportion of each of the classes is calculated. For the
+continuous raster map, the average values are calculated.
+
+<p>Once trained, the Random Forest model predicts weights in a raster
+grid. The spatial resolution of this output weighted grid should be
+specified via the <b>tile_size</b> parameter, and its name via the
+<b>output</b> parameter. The <b>tile_size</b> must be greater than
+the spatial resolution of <b>basemap_a</b>. It it considered good
+practice that the <b>tile_size</b> also be greater than the spatial
+resolution of the other basemaps and distance map. The extent of the
+output weighted grid is created using the extent of the spatial units,
+i.e. the <b>vector</b> parameter.
+
+<p>By default, all classes from the raster basemap(s) are taken into
+account. If the user want only specific classes to be taken into account
+within the Random Forest model, it is possible to provide an optional
+list of these classes with the <b>basemap_a_list</b> parameter (for
+basemap_a) or the <b>basemap_b_list</b> parameter (for basemap_b, if
+used).
+
+<p>The out-of-bag error (OOB) of the model is printed in the console
+and gives an indication about the internal accuracy of the model
+(cross-validation on training set at the spatial units level). The log
+of the Random Forest run (including the OOB error) is saved in a file
+whose path and name has to be specified via the <b>log_file</b>
+parameter. Using the <b>-f</b> flag, the log file will include extended
+information about the cross-validation for each set of parameters
+tested in the grid search.
+
+<p>Feature importances in the model are plotted in a graph, whose path
+and name has to be specified via the <b>plot</b> parameter. By default,
+class values are used. Optionally, class labels can be used as plot
+labels, using the flags <b>-a</b> and <b>-b</b>, for <b>basemap_a</b>
+and <b>basemap_b</b> respectively. If these flags are used, the raster
+should already contain the associated category labels. The module <a
+href="https://grass.osgeo.org/grass-stable/manuals/r.category.html">r.
+category</a> can be used to set category labels for a raster map. If the
+flag is selected, but the class label does not exist, the class value

```

Comment:
if the flag -a is selected?

Reply:
Well, -a and/or -b in fact. I will reformulate.